code/go/0chain.net/conductor/conductrpc/entity.go

```diff
@@ -29,6 +29,18 @@ func (e *Entity) State() (state *State) {
 	return e.state
 }
 
+// MagicBlock returns the location path of the magic block configuration.
+func (e *Entity) MagicBlock() (magicBlock string) {
+	magicBlockPtr, err := e.client.magicBlock()
+	if err != nil {
+		log.Fatalf("failed getting magic block: %v", err)
+	}
+
+	magicBlock = *magicBlockPtr
```

Comment:
if return unnamed parameter you don't need this line

Reply:
What do you mean?

Reply:
I got so used to using the named parameters that forgot the other way. Thanks

Reply:
fixed by [4922e7c](https://github.com/0chain/0chain/pull/1759/commits/4922e7cb226587a2e44f518f693f73ffc5a0d1b2)